ports/getdns/portfile.cmake

```
@@ -0,0 +1,46 @@
+vcpkg_fail_port_install(ON_TARGET "uwp")
+
+set(GETDNS_VERSION 1.6.0)
+set(GETDNS_HASH 4d3a67cd76e7ce53a31c9b92607d7768381a1f916e7950fe4e69368fa585d38dbfc04975630fbe8d8bd14f4bebf83a3516e063b5b54e0922548edc0952ee7b4e)
+
+string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "static" GETDNS_ENABLE_STATIC)
+string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" GETDNS_ENABLE_SHARED)
+
+vcpkg_download_distfile(ARCHIVE
+    URLS "https://getdnsapi.net/dist/getdns-${GETDNS_VERSION}.tar.gz"
+    FILENAME "getdns-${GETDNS_VERSION}.tar.gz"
+    SHA512 ${GETDNS_HASH}
+)
+
+vcpkg_extract_source_archive_ex(
+    OUT_SOURCE_PATH SOURCE_PATH
+    ARCHIVE ${ARCHIVE}
+    REF ${GETDNS_VERSION}
+    PATCHES
+        "openssl_depend_libs.patch"
+        "ignore_copying.patch"
+        "install_dlls.patch"
+)
+
+vcpkg_check_features(
+    OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    FEATURES
+        libevent BUILD_LIBEVENT2
+        libuv BUILD_LIBUV
```

```suggestion
    libevent BUILD_LIBEVENT2
    libuv BUILD_LIBUV
```
The key word `FEATURES `is not needed here in this case. Only when there are `INVERTED_FEATURES`, we should add it.

BTW, have you tested these two features?

---

If you say so. The documentation says it can be omitted, not that it must or even that omission is preferred. As it happens, I was following a port with INVERTED_FEATURES.

And yes, I have tested building both features. Each generates the appropriate additional library, and those libraries are not present unless you build with the feature, which is the expected behaviour.

---

You're right. It can be omitted, but not must. Thanks for your update.